src/main/java/com/siemens/ct/exi/core/context/GrammarUriContext.java

```diff
@@ -96,10 +97,7 @@ public QNameContext getQNameContext(String localName) {
 
 	protected static QNameContext binarySearch(QNameContext[] grammarQNames,
 			String localName) {
-		if (grammarQNames == null) {
-			System.err.println("ERROR null");
-		}
-		assert (grammarQNames != null);
+		Objects.requireNonNull(grammarQNames, "grammarQNames parameter could not be null");
```

Comment:
```suggestion
		Objects.requireNonNull(grammarQNames, "grammarQNames parameter cannot be null");
```

Reply:
Change is fine as is.. I would just not use "could" in this case...
Is that fine by you?